test/functional/apps/visualize/_heatmap_chart.js

```diff
@@ -49,20 +49,7 @@ export default function ({ getService, getPageObjects }) {
     });
 
     it('should save and load', async function () {
-      await PageObjects.visualize.saveVisualizationExpectSuccess(vizName1);
-      const pageTitle = await PageObjects.common.getBreadcrumbPageTitle();
-      log.debug(`Save viz page title is ${pageTitle}`);
-      expect(pageTitle).to.contain(vizName1);
-      await PageObjects.visualize.waitForVisualizationSavedToastGone();
-      await PageObjects.visualize.loadSavedVisualization(vizName1);
-      await PageObjects.visualize.waitForVisualization();
```

Comment:
This test is exactly the same as the next, minus the `await PageObjects.header.waitUntilLoadingHasFinished();`, so I assume it was accidentally duplicated during conflict resolution.